lib/searchkick/logging.rb

```diff
@@ -30,7 +30,7 @@ def store(record)
       end
     end
 
-    def remove(record)
+    def remove(record, options = {})
```

Comment:
We can remove the options here, right?

Reply:
i need to as its passing the arguments to its super